### ISSUE TITLE
Bug with ISO 8601 Datetime format

### ISSUE THREAD

**Issue by panikena** [okta/okta-sdk-dotnet#369]
Okta UI and Okta .NET SDK are inconsistent about the ISO 8601-formatted date. This issue was confirmed by Okta Support, ticked ID _00DA0KWYD._5001Y1OWUKr
Please see the following screenshots:
 - in Okta report page, I see my last login date as 08:57 UTC/GMT, which is correct.
![okta_ui](https://user-images.githubusercontent.com/18687071/79319288-cb958700-7f10-11ea-83d2-967f2e3729d1.png)
-  in the user entity that was returned by Okta .NET SDK I see that my last login date contains UTC/GMT DateTime component and EDT (-04:00) time offset, which is an offset of local client. I expect to see either 08:57 +00:00 or 04:57 -04:00, per [ISO 8601](https://www.iso.org/standard/70907.html) or [RFC3339](https://tools.ietf.org/html/rfc3339#section-4.2)

![okta_app](https://user-images.githubusercontent.com/18687071/79319481-0bf50500-7f11-11ea-8a3f-1f5f937c2d1b.png)

- however, the HTTP request that was sent  by the application to Okta API returned a correct date

![okta_fiddler](https://user-images.githubusercontent.com/18687071/79319798-77d76d80-7f11-11ea-9dbd-1a2bfc8242e0.png)

The affected package is Okta.Sdk 1.4.1, but from what I remember this issue was present in previous versions as well.

This causes significant confusion and necessity to write "hacks" that transform dates to correct format.

I'm looking forward to hearing from you!

**Comment by laura-rodriguez** [comment]
Hi @panikena,

Thanks for taking the time to report this issue, and for providing such detailed steps to reproduce it ❤️. We will review it and prioritize accordingly.

**Comment by laura-rodriguez** [comment]
Hi @panikena ,

This bug was fixed in [2.0.0](https://github.com/okta/okta-sdk-dotnet/releases/tag/v2.0.0).

However, you can still use the following workaround in 1.x series:
```
var serializer = new DefaultSerializer(new JsonSerializerSettings()
{
    DateParseHandling = DateParseHandling.None,
});

var client = new Okta.Sdk.OktaClient(new Okta.Sdk.Configuration.OktaClientConfiguration
{
    OktaDomain = "https://{yourOktaDomain}",
    Token = "{apiToken}"
}, serializer: serializer);

var user = await client.Users.GetUserAsync("user@test.com");
```
I believe you can use either  `DateParseHandling.None` or `DateParseHandling.DateTimeOffset`. Both should work.

Please let me know if it helps.
